src/commands/web/internal/spec/testFiles/script3.js

```diff
@@ -0,0 +1,248 @@
+// game.js from SASjs/Mario
+// https://github.com/sasjs/mario/blob/main/src/js/game.js
+
+var requestAnimFrame = (function () {
```

Comment:
should use const or let instead of var

Reply:
It's a test file for sasjs web application, copied from mario. Not part of cli codebase.